yarn.lock

```diff
@@ -27,7 +27,7 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
-"@babel/generator@^7.4.0", "@babel/generator@^7.4.4":
+"@babel/generator@^7.4.4":
```

Comment:
were these changes also made because of broken test:coverage? Should we break this out into a separate PR?

Reply:
I think so. sounds good to break into a separate PR.

Reply:
https://github.com/okta/okta-signin-widget/pull/1018